crates/biome_string_case/src/lib.rs

```diff
@@ -704,6 +713,10 @@ impl StrLikeExtension for std::ffi::OsStr {
         self.as_encoded_bytes()
             .ascii_nat_cmp(other.as_encoded_bytes())
     }
+
+    fn lexicographic_cmp(&self, other: &Self) -> Ordering {
+        self.as_encoded_bytes().cmp(other.as_encoded_bytes())
+    }
```

Comment:
How is this different from Rust's built-in `cmp`?

Reply:
As mentioned above.